### PR TITLE
OKTA-506935 Configurable Rate Limits by API Token

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added small updates to the Rate Limits overview and the Create an API Token pages of developer.okta.com on the feature configurable rate limits by API Token.
- **Is this PR related to a Monolith release?** 2022.07.0

### Resolves:

* [OKTA-506935](https://oktainc.atlassian.net/browse/OKTA-506935)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md [hunk 1]
```diff
@@ -47,14 +47,18 @@ To create an API token, follow these steps:
...
 
 3. Click **Create Token**.
 
 4. Name your token and click **Create Token**.
 
 5. Record the token value. This is the only opportunity to see it and record it.
 
+## Token rate limits
+
+When API tokens are created, rate limits for each token interaction are set automatically to 100 percent of each API maximum limit. See [API rate limits](/docs/reference/rate-limits/). You can adjust this percentage by editing each token. See [Set token rate limits](https://help.okta.com/okta_help.htm?id=ext_API#set-token-rate-limits).
```
Comment (leslielo-okta) [inline packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md:60]: This should be 50%

```suggestion
When API tokens are created, rate limits for each token interaction are set automatically to 50 percent of each API maximum limit. See [API rate limits](/docs/reference/rate-limits/). You can adjust this percentage by editing each token. See [Set token rate limits](https://help.okta.com/okta_help.htm?id=ext_API#set-token-rate-limits).
```

**Review by vanngo-okta — APPROVED** [review]
Looks good! Small nits, but not blocking the approval 👍

**Review by joec-okta — APPROVED** [review]
LGTM

**Review by leslielo-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md [hunk 2]
```diff
@@ -47,14 +47,18 @@ To create an API token, follow these steps:
...
 
 3. Click **Create Token**.
 
 4. Name your token and click **Create Token**.
 
 5. Record the token value. This is the only opportunity to see it and record it.
 
+## Token rate limits
+
+When API tokens are created, rate limits for each token interaction are set automatically to 50 percent of each API maximum limit. See [API rate limits](/docs/reference/rate-limits/). You can adjust this percentage by editing each token. See [Set token rate limits](https://help.okta.com/okta_help.htm?id=ext_API#set-token-rate-limits).
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md]: Added

**Pull request merged**